integration/bffsrv_test.go

```diff
@@ -75,6 +80,98 @@ func TestIngester(t *testing.T) {
 	assert.NotContains(t, postURIs, ignoredPost.Uri)
 }
 
+func postWithLangs(t *testing.T, u *indigoTest.TestUser, body string, langs []string) *atproto.RepoStrongRef {
+	t.Helper()
+
+	ctx := context.TODO()
+	resp, err := atproto.RepoCreateRecord(ctx, integration.ExtractClientFromTestUser(u), &atproto.RepoCreateRecord_Input{
+		Collection: "app.bsky.feed.post",
+		Repo:       u.DID(),
+		Record: &lexutil.LexiconTypeDecoder{
+			Val: &bsky.FeedPost{
+				CreatedAt: time.Now().Format(time.RFC3339),
+				Text:      body,
+				Langs:     langs,
+			},
+		},
+	})
+
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
Whilst I'm not the biggest fan of `require`/`assert`, I'd like to be consistent in us using them 

```suggestion
	require.NoError(t, err)
```